setup.cfg

```diff
@@ -1,5 +1,5 @@
 [tool:pytest]
-addopts = --pycodestyle --pydocstyle --mypy --black
+addopts = --mypy
```

Comment:
Was there something wrong with these flags?

Reply:
Somehow pytest wasn't accepting those flags--

"ERROR: usage: pytest [options] [file_or_dir] [file_or_dir] [...]
pytest: error: unrecognized arguments: --pycodestyle --pydocstyle --black"

I can bring them back for now and we can always work out what's happening with it during the meeting!